src/main/java/org/eclipse/xpanse/terraform/boot/models/providers/Aws.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
do not need this. Terraform-boot should have no information about xpanse or CSP, etc.
Terraform-boot is a generic project to execute any terraform scripts.